variables-extensions.tf

```diff
@@ -61,16 +61,42 @@ variable "calico_staging_dir" {
   type        = string
 }
 
+# CNI: Multus
+
+variable "multus_install" {
```

Comment:
I assume we can do both and not have to pick between the 2?

Reply:
Yes that's right.